### PR TITLE
 docs: change example in readme to slash command

### PR DESCRIPTION
**Pull request by SuperchupuDev**
**Please describe the changes this PR makes and why it should be merged:**
Since we are using application commands instead of message based commands for all examples, the readme should do it too for consistency

**Status and versioning classification:**
- Code changes have been tested against the Discord API, or there are no code changes
- I know how to update typings and have done so, or typings don't need updating
- This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

**Review by kyranet — APPROVED** [review]

**Review by SpaceEEC — APPROVED** [review]

**Review by vladfrangu — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -49,9 +49,11 @@ client.on('ready', () => {
   console.log(`Logged in as ${client.user.tag}!`);
```
Comment (ShockTr) [inline README.md:49]:
```suggestion
  console.log(`Logged in as ${client.user.tag}!`);
  if (await client.application.commands.fetch().then(commands => commands.size) === 0) {
            await client.application.commands.set([
                {
                    name: "ping",
                    description: "Replies with Pong!"
                }
            ])
        }
```
I think example code should also handle the creation of the slash command, here is a suggestion.
Note: You have to make the ready listener async for this suggestion to work

**Review by ShockTr — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (SuperchupuDev) [reply README.md:49]: talked about this with crawl, i'll add the deploy code of the "advanced deployment" section of the guide

File: README.md [hunk 2]
```diff
@@ -41,6 +41,32 @@ npm install discord.js
...
+
+  await rest.put(
+    Routes.applicationGuildCommands(CLIENT_ID, GUILD_ID),
+    { body: commands },
+  );
+
+  console.log('Sucessfully reloaded application (/) commands.');
+} catch (error) {
+  console.error(error);
+}
```
Comment (iCrawl) [inline README.md:67]:
```suggestion
(async () => {
	  try {
	    console.log('Started refreshing application (/) commands');
	  
	    await rest.put(
	      Routes.applicationGuildCommands(CLIENT_ID, GUILD_ID),
	      { body: commands },
	    );
	  
	    console.log('Sucessfully reloaded application (/) commands.');
	  } catch (error) {
	    console.error(error);
	  }
})
```

File: README.md [hunk 3]
```diff
@@ -41,6 +41,34 @@ npm install discord.js
 
 ## Example usage
 
+Execute this code first to deploy your command
```
Comment (iCrawl) [inline README.md:44]: First, we need to register a slash command against the Discord API:

File: README.md [hunk 4]
```diff
@@ -41,6 +41,34 @@ npm install discord.js
...
+      );
+    
+      console.log('Sucessfully reloaded application (/) commands.');
+    } catch (error) {
+      console.error(error);
+    }
+})()
+```
+Afterwards, run this to turn on your bot
 ```js
```
Comment (iCrawl) [inline README.md:72]:
```suggestion
```

Afterwards we can create a quite simple example bot:
```js
```

Comment (iCrawl) [reply README.md:72]: Wow I hate GitHub

File: README.md [hunk 5]
```diff
@@ -41,6 +41,34 @@ npm install discord.js
...
+        { body: commands },
+      );
+    
+      console.log('Sucessfully reloaded application (/) commands.');
+    } catch (error) {
+      console.error(error);
+    }
+})()
+```
+Afterwards, run this to turn on your bot
```
Comment (iCrawl) [inline README.md:71]:
```suggestion

Afterwards we can create a quite simple example bot:
```

Comment (iCrawl) [reply README.md:71]: Rather use this 😆

**Review by iCrawl — CHANGES_REQUESTED** [review]

**Review by iCrawl — APPROVED** [review]

**Pull request merged**